### PR TITLE
RELEASE v1.0.6

### PR DESCRIPTION
**Pull request by alinmigea**
ADD the plugin uses just data from ElasticSearch
ADD logs and updated change-logs
UPDATE the performance has been improved
UPDATE the price calls to point to ElasticExportPriceHelper

@plentymarkets/team-multichannel-koboldmaki

### CODE REVIEW

File: src/Generator/TracdelightCOM.php [hunk 1]
```diff
@@ -285,117 +320,101 @@ private function returnImagePath($variation, int $number)
      */
     private function getVariationAttributes($variation, KeyValue $settings):array
```
Comment (mariusbippig) [inline src/Generator/TracdelightCOM.php:322]: Isn't there a function in the EE helper to get the attributes?

File: src/Generator/TracdelightCOM.php [hunk 2]
```diff
@@ -285,117 +320,101 @@ private function returnImagePath($variation, int $number)
...
-                    // Get the color and size attribute value
-                    if(($attributeValueName->attributeValue->attribute->backendName == 'Color' || $attributeValueName->attributeValue->attribute->backendName == 'Size')
-                        && !is_null($attributeValueName->attributeValue->tracdelightValue))
+                    // Check if the attribute is available for Tracdelight
+                    if($attributeValueName->attributeValue->tracdelightValue)
                     {
-                        $variationAttributes[strtolower($attributeValueName->attributeValue->attribute->backendName)] = $attributeValueName->attributeValue->tracdelightValue;
+                        // Get the color and size attribute value
+                        if(($attributeValueName->attributeValue->attribute->backendName == 'Color' || $attributeValueName->attributeValue->attribute->backendName == 'Size')
+                            && !is_null($attributeValueName->attributeValue->tracdelightValue))
```
Comment (mariusbippig) [inline src/Generator/TracdelightCOM.php]: You have checked that value before

**Review by mariusbippig — CHANGES_REQUESTED** [review]

File: src/Generator/TracdelightCOM.php (hunk 1 above)
Comment (alinmigea) [reply src/Generator/TracdelightCOM.php:322]: At the moment there's no function to get the specific AttributeValue of an attribute, depending on the market dependency. We planned with Max to change that maybe into an AttributeHelper

File: src/Generator/TracdelightCOM.php (hunk 2 above)
Comment (alinmigea) [reply src/Generator/TracdelightCOM.php]: Modification done and I've added also two more type of attributes that I needed. I've retested right now and it works well.

File: src/Generator/TracdelightCOM.php (hunk 1 above)
Comment (mariusbippig) [reply src/Generator/TracdelightCOM.php:322]: ok!

**Review by mariusbippig — APPROVED** [review]

**Comment by mariusbippig** [comment]
Get the approve from Helena for the text stuff. After that you can directly merge and upload the new version =)

**Review by hschwab — APPROVED** [review]

**Pull request merged**